src/components/FeaturePanel/Climbing/RouteList/RouteListRow.tsx

```diff
@@ -86,7 +87,7 @@ export const RenderListRow = ({
       ref.current?.scrollIntoView({ behavior: 'smooth', block: 'nearest' });
     }
   }, [routeSelectedIndex]);
-  const osmId = route.feature?.osmMeta.id ?? null;
+  const osmId = getShortId(route.feature?.osmMeta) ?? null;
```

Comment:
Can you please check this? The `getShortId` fn expects only `OsmApiId` object, if you pass a null it will throw an exception.  (Can it really happen?) 

If no, the `?? null` is unnecessary then, check the other places as well.